packages/bundler/src/commands/inline.js

```diff
@@ -123,20 +126,25 @@ async function inlineSassDependencies(
 function findSassModule(packageName, cwd) {
   let currentDirectory = cwd;
 
-  while (currentDirectory !== '/') {
+  const isNotRoot = () =>
+    isWin
+      ? !/^[a-zA-Z]:(\/\/|\\)$/.test(currentDirectory)
+      : currentDirectory !== '/';
+
+  while (isNotRoot()) {
```

Comment:
Would the terminal case here be something like:

```js
while (currentDirectory !== path.dirname(currentDirectory)) {
  // ...
}
```

Basically just trying to say as long as moving up doesn't result in the same directory (root) 🤔

Reply:
Pretty smart way of checking that, will implement it this way 👍